api/v6/report_server.thrift

```diff
@@ -253,7 +263,8 @@ service codeCheckerDBAccess {
   // PERMISSION: PRODUCT_ACCESS
   RunHistoryDataList getRunHistory(1: list<i64> runIds,
                                    2: i64       limit,
-                                   3: i64       offset)
+                                   3: i64       offset,
+                                   4: RunHistoryFilter runHistoryFilter)
```

Comment:
Is extending a *function* signature with a new variable backwards compatible? (I know we have learnt extending records are.)

Reply:
Yes. E.g.: https://github.com/Ericsson/codechecker/pull/1094/files